pkg/virt-handler/vm.go

```diff
@@ -2528,6 +2520,18 @@ func (c *VirtualMachineController) checkVolumesForMigration(vmi *v1.VirtualMachi
 				return true, fmt.Errorf("cannot migrate VMI with non-shared HostDisk")
 			}
 		} else {
+			isVirtiofsFilesystem := false
```

Comment:
@germag that is the part I was mentioning offline where I'd like to see a unit test with filesystem + RWO/RWX pvc